cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/AbstractStreamFileAdmin.java

```diff
@@ -314,15 +314,15 @@ public void create(String name, @Nullable Properties props) throws Exception {
   }
 
   /**
-   * Create the public {@link NotificationFeed}s that concerns the stream with configuration {@code config}.
+   * Create the public {@link Id.NotificationFeed}s that concerns the stream with configuration {@code config}.
    *
    * @param config config of the stream to create feeds for
    */
   private void createStreamFeeds(StreamConfig config) {
     // TODO use accountID as namespace?
     try {
-      NotificationFeed streamFeed = new NotificationFeed.Builder()
-        .setNamespace(Constants.DEFAULT_NAMESPACE)
+      Id.NotificationFeed streamFeed = new Id.NotificationFeed.Builder()
+        .setNamespaceId(Constants.DEFAULT_NAMESPACE)
```

Comment:
Could we miss this while namespacing Streams though? Unsure if @anwar6953 is factoring changes here while namespacing streams. Please co-ordinate.